Makefile

```diff
@@ -592,7 +592,7 @@ localbenchmarks: test-binaries
 localsystem:
 	# Wipe existing config, database, and cache: start with clean slate.
 	$(RM) -rf ${HOME}/.local/share/containers ${HOME}/.config/containers
-	if timeout -v 1 true; then PODMAN=$(CURDIR)/bin/podman bats test/system/; else echo "Skipping $@: 'timeout -v' unavailable'"; fi
+	if timeout -v 1 true; then PODMAN=$(CURDIR)/bin/podman QUADLET=$(CURDIR)/bin/quadlet bats test/system/; else echo "Skipping $@: 'timeout -v' unavailable'"; fi
```

Comment:
Please make the same change here: https://github.com/containers/podman/blob/1b677c5202332ea7bb8753bd1fbe259c7323c991/hack/bats#L54-L56

The `hack/bats` script is super helpful to try out things locally.